passacre/config.py

```diff
@@ -301,12 +305,48 @@ def set_config(self, site, name, value):
             curs.execute(
                 'INSERT INTO config_values (site_name, name, value) VALUES (?, ?, ?)',
                 (site, name, jdumps(new_value)))
+        self.save()
+
+    def save(self):
         self._db.commit()
 
+class SqliteTahoeConfig(SqliteConfig):
+    @features.tahoe.check
+    def __init__(self):
+        try:
+            with open(os.path.expanduser("~/.tahoe/node.url")) as f:
+                self.tahoe = f.readline().strip()
+                self.tahoe += "uri/" if self.tahoe.endswith("/") else "/uri/"
+        except IOError:
+            self.tahoe = "http://localhost:3456/uri/"
+        super(SqliteConfig, self).__init__()
+
+    def read(self, uri):
+        '''Load site configuration from Tahoe-LAFS gateway'''
```

Comment:
The previous comment also applies to `save()`’s docstring, and [for consistency, always use `"""triple double quotes"""` around docstrings.][1]

  [1]: https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring